projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -74,8 +93,36 @@
     border: tokens.$cds-alias-object-border-width-100 solid datepicker-variables.$clr-calendar-border-color;
     border-radius: tokens.$cds-alias-object-border-radius-100;
     box-shadow: tokens.$cds-alias-object-shadow-100;
-    overflow: hidden;
     z-index: map.get(variables.$clr-layers, dropdown-menu);
+    justify-content: space-between;
+
+    clr-daypicker,
+    clr-monthpicker,
+    clr-yearpicker {
+      min-height: calc(100% - mixins.baselinePx(50));
+    }
```

Comment:
We are already setting min-height with a token for monthpicker and yearpicker. 'datepicker-variables.$clr-monthpicker-min-height'.  Add daypicker there and use `datepicker-variables.$clr-calendar-action-buttons-height` instead of hardcoded 50px